packages/typegpu/src/data/attributes.ts

```diff
@@ -105,7 +106,7 @@ export type Decorate<
       >
     : never;
 
-function attribute<
+export function attribute<
```

Comment:
why? this function doesn't seem to be useful in other files yet

Reply:
My bad. It is not useful in this PR, but it does get used by builtins in "the future" 🚀